uber/tests/models/test_attendee.py

```diff
@@ -251,11 +266,30 @@ def unset_volunteering(self, monkeypatch):
         return Attendee.unset_volunteering
 
     def test_dept_head_invariants(self):
-        a = Attendee(ribbon=c.DEPT_HEAD_RIBBON)
+        a = Attendee(ribbon=c.DEPT_HEAD_RIBBON, assigned_depts=c.CONSOLE)
         a._staffing_adjustments()
-        assert a.staffing and a.trusted
+        assert a.staffing
+        assert a.trusted_in(c.CONSOLE)
+        assert a.trusted_in_any_depts
         assert a.badge_type == c.STAFF_BADGE
 
+    def test_trust_TrustedInAssignedDept_AfterStaffingAdjustments(self):
+        a = Attendee(staffing=True,
+                     assigned_depts='{},{}'.format(c.CONSOLE, c.CON_OPS),
+                     trusted_depts='{},{}'.format(c.CONSOLE, c.CON_OPS))
+        a._staffing_adjustments()
+        assert a.assigned_to(c.CONSOLE) and a.trusted_in(c.CONSOLE)
+        assert a.assigned_to(c.CON_OPS) and a.trusted_in(c.CON_OPS)
+
+    def test_trust_NotTrustedInAnUnassignedDept_AfterStaffingAdjustments(self):
+        a = Attendee(staffing=True,
+                     assigned_depts='{},{}'.format(c.CONSOLE, c.CON_OPS),
+                     trusted_depts='{},{}'.format(c.ARCADE, c.CON_OPS))
+        a._staffing_adjustments()
+        assert a.assigned_to(c.CONSOLE) and not a.trusted_in(c.CONSOLE)
+        assert not a.assigned_to(c.ARCADE) and not a.trusted_in(c.ARCADE)
+        assert a.assigned_to(c.CON_OPS) and a.trusted_in(c.CON_OPS)
+
```

Comment:
one thing to note on these new tests, I decided to try and name them something more descriptive to what the test is doing.  came across this here:
http://osherove.com/blog/2005/4/3/naming-standards-for-unit-tests.html

lemme know if that camel case is making your eyes bleed and we can try and swap for something else